openedx/core/djangoapps/credit/api/provider.py

```diff
@@ -23,6 +23,10 @@
     CreditRequest,
     CreditEligibility,
 )
+
+from student.models import (
```

Comment:
The from student.models import User is already defined on line 31. Can you move User inside this tuple?